packages/paste-core/primitives/text/src/types.ts

```diff
@@ -14,19 +7,19 @@ import type {
   SpaceProps,
   TypographyProps,
   VerticalAlign,
+  CursorProperty,
+  OutlineProperty,
+  TransitionProperty,
 } from '@twilio-paste/style-props';
 import type {PseudoPropStyles} from './PseudoPropStyles';
 
 export interface TextBaseStyleProps extends OverflowProps, PositionProps, ShadowProps, SpaceProps, TypographyProps {
   content?: string;
   cursor?: CursorProperty;
   display?: Display;
-  outline?: OutlineProperty<string>;
-  // Do not document, we prefer if folks do not use this property for i18n.
-  textTransform?: TextTransformProperty;
+  outline?: OutlineProperty;
   transition?: TransitionProperty;
   verticalAlign?: VerticalAlign;
-  fontVariantNumeric?: FontVariantNumericProperty;
```

Comment:
`textTransform` was a duplicate already declared on `TypographyProps`

`fontVariantNumber` moved to `TypographyProps`